assets/sass/theme_overwrites.scss

```diff
@@ -118,6 +118,10 @@ a {
   @include transition(all 0.2s);
 }
 
+abbr[title], acronym[title] {
+  text-decoration: none;
+}
```

Comment:
we (bootstrap) set(s) our own dotted underline via border-bottom as chrome doesn't seem to support dotted underlines. This currently results in a single underline in chrome and a double underline in firefox.
This change will remove the double dotted line in firefox.